### PR TITLE
add guide for v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Review by iulianatodoran — APPROVED** [review]
The Upgrade Guide for 1.8.0 LGTM

**Comment by iulianatodoran** [comment]
> For Java, similarly to Python, you can add:
> 
> * We changed the name of the Maven package from `altunityrunner` to `AltTester-Driver`. After downloading the package to install the jar run mvn install:install-file -Dfile=./target/AltTester.jar -DgroupId=com.altom -DartifactId=alttester -Dversion=1.8.0 -Dpackaging=jar``
> * We renamed the `altunityrunner` jar to `alttester`, replace all import from `import altunityrunner.*`   to `import alttester. *`.

The correct jar package name is `AltTester-Driver.jar`

File: Docs/source/pages/upgrade-guides/v-1-8-0.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    
+Java
+----
+**altunityrunner**  →  **alttester**
+
+1. In case of `altunityrunner` Maven package, we changed the name to `AltTester-Driver`.
+
+To install the jar run ``mvn install:install-file -Dfile=./target/AltTester-Driver.jar -DgroupId=com.alttester -DartifactId=alttester -Dversion=1.8.0 -Dpackaging=jar``.
+
+2. In case of  `altunityrunner` jar, we changed the name to `alttester`.
```
Comment (gomboskriszta15) [inline Docs/source/pages/upgrade-guides/v-1-8-0.rst:83]: The jar was named AltTester-Driver in the end...

File: Docs/source/pages/upgrade-guides/v-1-8-0.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+Replace:
+
+.. code-block:: Java
+
+    import altunityrunner.*
```
Comment (gomboskriszta15) [inline Docs/source/pages/upgrade-guides/v-1-8-0.rst:89]: From `import ro.altom.atlunitytester.*` changed to `import com.alttester.*`.

**Review by gomboskriszta15 — APPROVED** [review]
👍

**Pull request merged**